Detectors/PHOS/base/include/PHOSBase/Geometry.h

```diff
@@ -85,9 +85,9 @@ class Geometry
   const std::string& GetName() const { return mGeoName; }
 
  private:
-  static Geometry* sGeom; ///< Pointer to the unique instance of the singleton
+  static Geometry* sGeom; // Pointer to the unique instance of the singleton
```

Comment:
please leave the `///<` in as this is required by doxygen to interpret the comment as brief description